go/vt/vttablet/tabletserver/schema/db.go

```diff
@@ -86,6 +88,32 @@ where table_schema = database() and table_name in ::viewNames`
 
 	// fetchTablesAndViews queries fetches all information about tables and views
 	fetchTablesAndViews = `select table_name, create_statement from %s.tables where table_schema = database() union select table_name, create_statement from %s.views where table_schema = database()`
+
+	// detectUdfChange query detects if there is any udf change from previous copy.
+	detectUdfChange = `SELECT name
+FROM (
+	SELECT name FROM 
+	mysql.func 
+
+	UNION ALL
+
+	SELECT function_name
+	FROM %s.udfs
+) _inner
+GROUP BY name
+HAVING COUNT(*) = 1
+LIMIT 1
```

Comment:
The way I'm reading this, we'll detect new or deleted UDFs, but not a change to the UDF definition?

Reply:
Good point, we should also check the types